examples/pytorch/graphsage/train_sampling.py

```diff
@@ -234,11 +248,6 @@ def run(args, device, data):
         train_nfeat = train_nfeat.to(device)
         train_labels = train_labels.to(device)
 
-    # Create csr/coo/csc formats before launching training processes with multi-gpu.
-    # This avoids creating certain formats in each sub-process, which saves momory and CPU.
-    train_g.create_formats_()
-    val_g.create_formats_()
-    test_g.create_formats_()
```

Comment:
Since this is a single-gpu example, creating these formats before-hand is unnecessary.

Reply:
Exactly.